src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final int figures = 5;
+    private final int maxSize = 99;
```

Comment:
It should be constants

Reply:
What do you mean? Maybe these constants should't be fields?